tests/test_flow/test_flow__purge_page_view_data.py

```diff
@@ -0,0 +1,197 @@
+from __future__ import division
```

Comment:
Two underscores in file name intentional?

Reply:
Yes. But since it's under the `test_flow` folder (flow is huge module to test), I think it's not necessary. Changed to `test_purge_page_view_data.py`